drivers/pwm/Kconfig.esp32

```diff
@@ -8,3 +8,9 @@ config PWM_LED_ESP32
 	help
 	  This option enables the PWM LED driver for ESP32 family of
 	  processors. Say y if you wish to use PWM LED port on ESP32.
+
+config MCPWM_ESP32
```

Comment:
please enable based on dt compat status being enabled (see e.g. gd32), and make it dependent on ESP32 SoCs. I can now select both ESP32 drivers even if compiling for other platforms... this needs to be fixed.

Reply:
OK